rescript

```diff
@@ -15,20 +15,13 @@ var bsc_exe = require("./scripts/bin_path").bsc_exe;
 var rescript_exe = require("./scripts/bin_path").rescript_exe;
 var bsconfig = "bsconfig.json";
 
-var LAST_BUILD_START = 0;
-var LAST_FIRED_EVENT = 0;
-/**
- * @type {[string,string][]}
- */
-var reasonsToRebuild = [["proj", "started"]];
```

Comment:
I moved almost all `build -w` related code under the `if (maybeSubcommand === "build" && process_argv.includes("-w")) {` condition.